cmd/statusd/utils.go

```diff
@@ -1414,6 +1418,29 @@ func testJailFunctionCall(t *testing.T) bool {
 	return true
 }
 
+func testNodeOffline(t *testing.T) bool {
+
+	StopNode()
+	//time to sync
+	time.Sleep(5 * time.Second)
```

Comment:
Guys, please, familiarise yourself with the scope of the second milestone: https://github.com/status-im/status-go#boards?milestones=Beta%20milestone%202%20of%206%232017-08-25&notFullScreen=false&repos=61047867

There's currently no more reliable way and this is why: https://github.com/status-im/status-go/issues/246

Fixing synchronisation reliability is out of scope of @MarinX's task.